src/subapps/dataExplorer/DataExplorer.tsx

```diff
@@ -115,6 +125,7 @@ export const DataExplorer: React.FC<{}> = () => {
         pageSize={pageSize}
         offset={offset}
         updateTableConfiguration={updateTableConfiguration}
+        showEmptyDataCells={showEmptyDataCells}
```

Comment:
this is related to line `119` `columnsFromDataSource(currentPageDataSource, showMetadataColumns, selectedPath)`
I think it's better to wrap it in useCallback since the columns are calculated and it's does have to be always related to other part of the UI